.vscode/settings.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Here is a link https://github.com/github/gitignore/blob/main/Global/VisualStudioCode.gitignore to a repository that has useful gitignore values for different languages and editors. In this file this directory is actually excluded :)